src/main/scala/prv/saevel/scala/oop/cars/RentalService.scala

```diff
@@ -8,10 +8,10 @@ class RentalService(implicit carRepository: CarRepository, personRepository: Per
     val carOption = carRepository.findById(carId)
     var isFree:Boolean = true
       for(car<-carOption){
-      if (car.isFree ==true){
-        isFree =true
-      }else{
-        isFree =false
+      if (car.isFree){
```

Comment:
@Saevel is there more "elegant" and/or direct way of accessing data stored in object to which reference we have or you have to iterate through all available ones?